pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "poetry-core"
-version = "1.1.0a6"
+version = "1.1.0-alpha.7"
```

Comment:
Is there a reason why the naming pattern is changed? And why is it changed here, but not in `__init__.py`?

Reply:
Good point :+1: I bumped the version via `poetry version` and this uses SemVer nomenclature in the pyproject.toml. I guess this was done before manually. 

I've change the `__init__.py` file to read the version now via importlib. So we can avoid duplicate places for the version number.

Reply:
> I've change the __init__.py file to read the version now via importlib. So we can avoid duplicate places for the version number.

hm, seems to be problematic, because poetry-core needs to build itself. Will revert it.

Reply:
OK, but both "versions" should at least be equal, shouldn't they?

Reply:
Isn't this pep440 compliant? Just expanded?

Reply:
While reading the [part about normalization](https://www.python.org/dev/peps/pep-0440/#normalization), I thought the same: "seems to be valid, just not normalized". However, the [part about compatibility with semantic versioning](https://www.python.org/dev/peps/pep-0440/#semantic-versioning) says _"Semantic versions containing a hyphen (pre-releases - clause 10) or a plus sign (builds - clause 11) are not compatible with this PEP and are not permitted in the public version field."_

Maybe, versions which are not normalized are not considered PEP440 compliant but only have to be tolerated for compatibility reasons?

Reply:
Or, you could avoid all this confusion by releasing plain old 1.1.0...  

What is the reason for alpha releases of poetry-core?  Just ship what's good now, and ship fixes / additions / changes in 1.1.1 / 1.2.0 / 2.0.0.  There's no shortage of version numbers.

(Possibly a conversation for another place, but I'd encourage a poetry 1.2.0 release too... do point me at it if this is being discussed somewhere.  There are lots of fixes stored up in master that are never going to get back-applied to the 1.1 branch, let's release them)

Reply:
I understand PEP-440 in that way that `-alpha.7` is compliant in that way, that parser have to accept it, but it's not allowed to write :man_shrugging: 

@dimbleby: I'm not sure at the moment if something is missing or to unstable to release a 1.1.0 of poetry-core. A poetry 1.2 is definitiv not possible yet, because the plugin system is not enough stable.

So @python-poetry/core: Is there something that should be changed here now or can we go on releasing?

Reply:
> I'm not sure at the moment if something is missing or to unstable to release a 1.1.0 of poetry-core

It's complete / stable enough to be used by `poetry` - as demonstrated by its master branch.  I don't see users from any other projects asking for additions or fixes.

> A poetry 1.2 is definitely not possible yet, because the plugin system is not enough stable.

Then my suggestion would be to cut a release, but declare that the plugin system is not yet stable.  

(Unless there's some work to bring that desired stability which is nearing completion, in which case it might be worth waiting for it.  But I get the impression that we are likely to be waiting for some time, and it's a shame to let the perfect be the enemy of the good.)

Reply:
We discussed it whether we want to call the release 1.1.0. We decided to keep it as an alpha release until we are sure enough that there are no more big changes needed before the poetry 1.2.0 release.